src/main/java/frc/robot/subsystems/ArmSubsystem.java

```diff
@@ -11,43 +20,32 @@
 
 public class ArmSubsystem extends SubsystemBase implements Loggable {
     public enum State {
-        Pickup, Rest, PlacingA, PlacingB, PlacingC, PlacingD, PlacingE, Transit // Possible System States
-    }
-
-    private final double elbowPosToDegrees = 599.9;
-    private final double shoulderPosToDegrees = 342.8;
-    private final double stateTolerance = 2;
-
-    // System motor controllers
-    private final CANSparkMax m_shoulder = new CANSparkMax(ArmConstants.SHOULDER_ID, MotorType.kBrushless);
-    private final CANSparkMax m_shoulderFollower = new CANSparkMax(ArmConstants.SHOULDER_FOLLOWER_ID, MotorType.kBrushless);
-    private final CANSparkMax m_elbow = new CANSparkMax(ArmConstants.ELBOW_ID, MotorType.kBrushless);
-
-    // System State - init in rest state
-    private State m_state_desired = State.Rest;
-    private State m_state_actual = State.Rest;
+        Pickup(0, 0), Rest(0, 0), PlacingA(Math.PI / 4, 0), PlacingB(0, 0), PlacingC(0,

```

Comment:
Fix this whitespace if we're using this. Each value should be a different line.

Reply:
Sorry, my formatter did the by itself.